design_handoff_mission_control/prototype/v2-bold.jsx

```
@@ -0,0 +1,624 @@
+                  fontFamily:V2.font, border:`1px solid ${V2.border}`, background:V2.surface, color:V2.ink, outline:'none',
+                }}
+                onFocus={(e) => e.target.style.borderColor = V2.amber}
+                onBlur={(e) => e.target.style.borderColor = V2.border}
+              />
+            </div>
+          );
+        })}
+
+        {/* Insight */}
+        <div style={{ marginTop:8, padding:'12px 14px', border:`1px solid ${V2.border}`, background:V2.surface }}>
+          <MonoLabel color={V2.amber} style={{ marginBottom:6 }}>// PATTERN · LAST 7</MonoLabel>
```

ESLint

[error] Comments inside children section of tag should be placed inside braces

(react/jsx-no-comment-textnodes)